### ISSUE TITLE
BUG: Prototype doesn't show all permits available, as promised on first page

### ISSUE THREAD

**Issue by petegale** [alphagov/verify-local-patterns#515]

**Comment by suryanaga** [comment]
So I think the place that should route people to business and trade permits is the page titled "Are you a resident?"

I don't know what the content for the radio button options should be though. What do you think @lizziebruce?

<img width="925" alt="screen shot 2017-05-16 at 15 16 51" src="https://cloud.githubusercontent.com/assets/4106955/26110637/d82e93fc-3a4a-11e7-9a9b-dc7efc5a4357.png">

**Comment by lizziebruce** [comment]
@sanjaypoyzer @EUzkuraityte couple of questions

1. is it ever the case that a visitor (guest) would apply for their own parking permit? Doesn't the resident apply for a visitor permit for their guest? Is this something you think needs to change about the way the service is run @sanjaypoyzer?

2. are there any more types permits users can apply for via this service @EUzkuraityte? So far we have resident's, visitor's, trade/business.

**Copy feedback**
Dependent on answer to 1., change to "I'm a guest of a resident." or "Yes but I'm applying for a guest"– but this is an inelegant solution so please see new copy version at bottom of this comment.
"I run a business" also needs to be changed to something like "I work in Argleton", as you can get a trade parking permit without being the director of the business. 

**Evidence vvv**
Wandsworth has a good definition:
Trade permits are available to those who are working as trades people, e.g. plumbers, builders etc, at a property within a Controlled Parking Zone (CPZ) which is not their permanent place of employment. 

And Oxford has:
Permits are for use in the vehicles owned by or hired or leased to a business or registered to employees of the business...

Therefore I'd rephrase this whole section as below. @EUzkuraityte please advise on whether we need to clarify type of work further based on the pilot participants.

> I need to park in Argleton because:
> 
> O I live here
> 
> O I work here
> 
> O I'm visiting someone who lives here

or

> I need a permit to park in Argleton because:
> 
> O I live here
> 
> O I live here, and have a visitor
> 
> O I work here

or

> I need a permit to park in Argleton for:
> 
> O Myself, I live here
> 
> O Myself, I work here 
> 
> O A visitor – I live here

**Comment by EUzkuraityte** [comment]
@lizziebruce 

visitors would never apply for visitor permits, only residents can do it and then distribute to their visitors as they see fit. I believe the reason behind it is that these two types of permits allow to park in residential parking bays, whilst city visitors should not be using those and park in the designated bays or car parks. 

Here is the info from Pete re: permit types offered by councils: 'We know there is a need to apply for residents and visitors permits at the same time. We also know that where permit limits allow it (especially with visitors scratch cards) there is a need to buy more than one at a time. 

Text file attached contains all the variations in residential and resident permit types and zones for participating councils
[permit types.txt](https://github.com/alphagov/verify-local-patterns/files/903263/permit.types.txt)'
Users can usually apply for a variation of these types of permits as well (the selection tends to depend on the borough):

Medical
Carer's
Business / Trader
Doctor
Teacher
Clergy 
Church visitor
Car Club
Funeral waivers
Market traders

Versions 2 and 3 look correct to me, probably 2 looks better. I think 'work bit' needs a little bit of tweaking as, depending on a council,  business permits can be applied for:
 - zones that you live in (in the area or CPZ) and, therefore, need to park your work / business car in the residential bay next to your home.
 - if the business address is within the CPZ
- some can be vehicle specific, some transferable, 
 - etc...

I'd say let's not spend more time on this, as we'll never be able to capture all possible reasons for why people apply for a business permit.. If you can make it more generic and leave it up to councils to amend or add an appropriate option, that'd be ok.

**Comment by lizziebruce** [comment]
@EUzkuraityte OK shall we use the 2nd copy suggestion (pasted below for clarity) in the prototype then with the assumption that individual councils can tailor the subsequent routing steps to reflect the full range of visitor/work permits available from that particular council, and leave the user to choose between live/work reasons as most appropriate, if it is for the same space (as I've understood from your response)?

> I need a permit to park in Argleton because:
> 
> O I live here
> 
> O I live here, and have a visitor
> 
> O I work here

...After further thought: to make the preferred copy fit better with the page header copy, amend to:

> Choose the best description of your parking requirements:
> 
> O I live here
> 
> O I live here, and have a visitor
> 
> O I work here

**Comment by EUzkuraityte** [comment]
linking to #584 #294
